griptape/drivers/image_generation/base_multi_model_text_to_image_generation_driver.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import annotations
+
+from attr import field, define
+
+from griptape.drivers import BaseImageGenerationDriver, BaseImageGenerationModelDriver
+
+
+@define
+class BaseMultiModelTextToImageGenerationDriver(BaseImageGenerationDriver):
```

Comment:
Is this still necessary with `BaseMultiModelImageGenerationDriver`?

Reply:
It is not; removed